test/throw.test.js

```diff
@@ -74,6 +74,18 @@ test('Fastify should throw for an invalid schema, printing the error route - bod
   })
 })
 
+test('Fastify should throw for an invalid shorthand option type', t => {
```

Comment:
We should add a test where shortHand is false and the user defines a route with that syntax and fastify doesn't start

Reply:
What do you mean? If you mean defining an invalid schema, it will just take it as an "accept all", fastify would still start just fine.

Reply:
Oh, do you mean under the @fastify/ajv-compiler-8 tests? I just saw that, I suppose that would be the way to create the failure on invalid schemas. I'll see how that plays together.

Reply:
👍 Figured it out. Added the test in latest push.